docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -475,8 +475,6 @@ The full example from this guide is available at https://github.com/expo/eas-tes
 
```

Comment:
In https://github.com/expo/eas-tests-example/pull/5/files#diff-345eed02a6bbda5de8c41d7648082bf758b59796089610e1e55b7d507ca9bb0f, you changed `.detoxrc.json`. The file is shown in this doc (lines 139-183). Please update it.